src/test/java/uk/gov/hmcts/reform/professionalapi/service/impl/ProfessionalUserServiceImplTest.java

```diff
@@ -631,7 +636,7 @@ public void checkUserStatusIsActiveByUserId_Throws403_WhenNoUserFoundWithGivenId
 
     @SneakyThrows
     @Test(expected = AccessDeniedException.class)
-    public void checkUserStatusIsActiveByUserId_Throws403_WhenUserIsNotActive() {
+    public void test_checkUserStatusIsActiveByUserId_Throws403_WhenUserIsNotActive() {
         professionalUser.setIdamStatus(IdamStatus.PENDING);
 
         when(professionalUserRepository.findById(any(UUID.class))).thenReturn(Optional.of(professionalUser));
```

Comment:
add verify for when condition

Reply:
add verify conditions for all when conditions in test cases in the ProfessionalUserServiceImplTest.java class

Reply:
verify doesn't work here. Once flow goes to actual method, it throws exception and it is caught by 'expected' annotation defined above test method.
Adding verify to all methods of this class should be covered in tech debt as it is not related to current code changes.

Reply:
> add verify conditions for all when conditions in test cases in the ProfessionalUserServiceImplTest.java class

ok.

Reply:
> verify doesn't work here. Once flow goes to actual method, it throws exception and it is caught by 'expected' annotation defined above test method.
your statement is not correct..if test contains when condition  100 % verify condition must satisfy otherwise test is not required when condition .

Reply:
Ok then please add one verify condition in this case after actual method call and let me know if code reaches to verify condition.if it reaches then I will add the verify condition.

Reply:
ok. updated